sdk/core/azure-core/azure/core/tracing/abstract_span.py

```diff
@@ -43,28 +43,29 @@ def finish(self):
         pass
 
     @abstractmethod
-    def to_header(self, headers):
+    def to_header(self):
         # type: (Dict[str, str]) -> Dict[str, str]
         """
         Returns a dictionary with the header labels and values.
         """
         pass
 
+    @property
     @abstractmethod
-    def from_header(self, headers):
-        # type: (Dict[str, str]) -> Any
+    def span_instance(self):
+        # type: () -> Any
```

Comment:
Wouldn't this return `AbstractSpan`?

Reply:
The span instance would return whatever the implemented type is and not the Abstract Span.

Reply:
Would that inherit `AbstractSpan`?

Reply:
nope